sunpy/net/dataretriever/sources/ace.py

```diff
@@ -0,0 +1,336 @@
+#This module was developed with funding provided by
+#the Google Summer of Code 2016.
+
+__author__ = "Sudarshan Konge"
+__email__ = "sudk1896@gmail.com"
+
+import datetime
+import urllib2
+
+
+from sunpy.net.dataretriever.client import GenericClient
+from sunpy.util.scraper import Scraper
+from sunpy.time import TimeRange
+
+__all__ = ['SWEPAMClient', 'EPAMClient', 'MAGClient', 'SISClient']
+
+class SWEPAMClient(GenericClient):
+    """
+    Returns a list of URLS to ACE SWEPAM files corresponding to value of input timerange.
+    URL source: `ftp://ftp.swpc.noaa.gov/pub/lists/ace/`.
+    The earliest date available is from 29-Jul-2015
+    Parameters
+    ----------
+    timerange: sunpy.time.TimeRange
+        time range for which data is to be downloaded.
+        Example value - TimeRange('2015-12-30 00:00:00','2015-12-31 00:01:00')
+    
+    Instrument: Fixed argument = 'swepam'
+    
+    Returns
+    -------
+    urls: list
+          list of urls corresponding to requested time range.
+    Examples
+    --------
+    >>> from sunpy.net import Fido
+    >>> from sunpy.net import attrs as a
+    >>> results = Fido.search(a.Time('2015/12/28 00:00:00', '2015/12/28 00:03:00'), a.Instrument('swepam'))
+    >>> print(results)
+    >>> [<Table length=3>
```

Comment:
This is not right, `>>>` should not be in the printed results. I believe I've seen this in some previous PR